src/main/java/co/cask/hydrator/plugin/DateTransform.java

```diff
@@ -181,25 +186,33 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
     @Nullable
     private String secondsOrMilliseconds;
 
+    @Name("ignoreErrors")
+    @Description("Ignore any errors in processing. If error dataset is used, the errors will not be ignored")
```

Comment:
This description should be more exact about what ignore means. Does ignore mean the input record is emitted as an error record? Or does it mean the target field value is set to null? Or does it mean something else?